frontend/src/app/modules/lessons/lessons-create/lessons-create.component.html

```diff
@@ -1,56 +1,121 @@
+                </div>
+
+                <img src="assets/interestsDropdown/chevron_down.svg" alt="chevron_down-icon" class="timeDropdown">
+                <div *ngIf="levelDropdownVisible" class="dropdownContent">
+                    <p *ngFor="let level of levelsList" (click)="updateLevel($event)">{{level}}</p>
+                </div>
+            </div>
+
+            <mat-error *ngIf="!level && submitted">
+                Level is Required
             </mat-error>
-        </mat-form-field>
+        </div>
 
-        <mat-form-field>
-            <mat-label>Tags</mat-label>
-            <mat-select placeholder="Select lesson tags" formControlName="tags" multiple>
-                <mat-option *ngFor="let tag of tagsList" [value]="tag">{{ tag }}</mat-option>
-            </mat-select>
+        <div class="form-field">
+            <app-interests-dropdown [usedInModal]="true" (selectedInterests)="updateTags($event)"></app-interests-dropdown>
 
-            <mat-error *ngIf="tags?.invalid && tags?.touched">
-                At least one tag is required
+            <mat-error *ngIf="!tagsList.length && submitted">
+                At least 1 Interest is Required
+            </mat-error>
+        </div>
+
+        <div class="form-field">
+            <div class="input">
+                <img src="assets/lesson-icons/people-icon.svg" alt="people-icon" class="icon">
+                <input type="number" [min]="1" placeholder="Number of students" formControlName="studentsCount">
+            </div>
+
+            <mat-error *ngIf="!studentsCount?.value && studentsCount?.touched ">
+                Number of students is Required
+            </mat-error>
+            <mat-error *ngIf="studentsCount?.value && studentsCount?.value < 1 && studentsCount?.touched ">
+                Number of students cannot be less than 1
+            </mat-error>
+        </div>
+
+        <div class="form-field">
+            <div class="input">
+                <img src="assets/lesson-icons/chains.svg" alt="chains-icon" class="icon">
+                <input type="text" placeholder="Zoom Link to the Lesson" formControlName="meetLink">
+            </div>
+
+            <mat-error *ngIf="meetLink?.invalid && meetLink?.touched">
+                Meet Link is Required
+            </mat-error>
+        </div>
+
+        <div class="form-field">
+            <textarea placeholder="Questions for the Group Class" formControlName="questions"></textarea>
+
+            <mat-error *ngIf="questions?.invalid && questions?.touched">
+                At least one question is required
             </mat-error>
-        </mat-form-field>
+        </div>
 
-        <button mat-raised-button [disabled]="myForm.invalid" color="primary" (click)="createLesson()">Create a lesson</button>
+        <!-- TODO: Update create button when design is finished -->
 
+        <button mat-flat-button color="primary" (click)="createLesson()">SAMPLE CREATE BUTTON</button>
```

Comment:
Change to create lesson
If we won't have it in design, we won't need to fix it at least